src/screens/TopicPageScreen/index.js

```diff
@@ -324,76 +361,6 @@ const TopicPageScreen = (props) => {
     }
   };
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `showHeaderAnimation` function and related logic have been removed. Ensure that the removal of this function does not affect the functionality of the component.
```diff
-  const showHeaderAnimation = () => {
-    interactionManagerRef.current = InteractionManager.runAfterInteractions(() => {
-      Animated.timing(animatedHeight, {
-        toValue:
-          (coverPath
-            ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER
-            : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT) +
-          dimen.size.TOPIC_FEED_HEADER_HEIGHT +
-          topPosition +
-          normalize(4),
-        duration: 100,
-        useNativeDriver: false
-      }).start();
-      Animated.timing(opacityHeaderAnimation, {
-        toValue: 1,
-        duration: 100,
-        useNativeDriver: false
-      }).start();
-    });
-    setIsHeaderHide(false);
-  };
-
-  React.useEffect(() => {
-    showHeaderAnimation();
-  }, [coverPath]);
-
-  React.useEffect(() => {
-    const unsubscribe = navigation.addListener('focus', () => {
-      showHeaderAnimation();
-    });
-
-    return () => {
-      if (interactionManagerRef.current) interactionManagerRef.current.cancel();
-      if (interactionManagerAnimatedRef.current) interactionManagerAnimatedRef.current.cancel();
-      unsubscribe();
-    };
-  }, [navigation, coverPath]);
-
-  const handleOnScrollBeginDrag = (event) => {
-    lastDragY = event.nativeEvent.contentOffset.y;
-  };
-
-  const handleScrollEvent = React.useCallback(
-    (event) => {
-      const {y} = event.nativeEvent.contentOffset;
-      const dy = y - lastDragY;
-      if (y <= 30) {
-        showHeaderAnimation();
-      } else if (
-        dimen.size.TOPIC_CURRENT_ITEM_HEIGHT &&
-        dy - 20 > dimen.size.TOPIC_CURRENT_ITEM_HEIGHT / 2
-      ) {
-        interactionManagerAnimatedRef.current = InteractionManager.runAfterInteractions(() => {
-          Animated.timing(animatedHeight, {
-            toValue: dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2 + topPosition,
-            duration: 100,
-            useNativeDriver: false
-          }).start();
-          Animated.timing(opacityHeaderAnimation, {
-            toValue: 0,
-            duration: 100,
-            useNativeDriver: false
-          }).start();
-        });
-        setIsHeaderHide(true);
-      }
-    },
-    [animatedHeight, coverPath]
-  );
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->